### PR TITLE
DOC-643: Added documentation for new listbox dialog UI component

### PR DESCRIPTION
**Pull request by lnewson**
Related Ticket: DOC-643

Description of Changes:
* Adds documentation for the new `listbox` component.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by SimonFc — APPROVED** [review]

File: _config.yml [hunk 1]
```diff
@@ -52,6 +52,7 @@ requires_5_1v: "> **Note**: This feature is only available for TinyMCE 5.1 and l
 requires_5_2v: "> **Note**: This feature is only available for TinyMCE 5.2 and later."
 requires_5_3v: "> **Note**: This feature is only available for TinyMCE 5.3 and later."
 requires_5_4v: "> **Note**: This feature is only available for TinyMCE 5.4 and later."
+requires_5_5v: "> **Note**: This feature is only available for TinyMCE 5.5 and later."
```
Comment (tylerkelly13) [inline _config.yml:55]:
```suggestion
```

Comment (tylerkelly13) [reply _config.yml:55]: Added on https://github.com/tinymce/tinymce-docs/pull/1673/files

**Review by tylerkelly13 — APPROVED** [review]
LGTM, would you like me to do the Release note?

File: _config.yml (hunk 1 above)
Comment (lnewson) [reply _config.yml:55]: This shouldn't be needed, as it'll detect it's the same change and just merge them.

**Review by ashfordneil — APPROVED** [review]

**Comment by tylerkelly13** [comment]
Sorry for force push, added release note for wrong setting :facepalm:

**Review by EkimChau — APPROVED** [review]

**Pull request merged**